Jenkinsfile

```diff
@@ -261,17 +264,34 @@ pipeline {
                     }
 
                     def testReportsUrl = getLogsURL(logServer, env.ZUUL_UUID)
-
                     if (isGithub()) {
                         sendGithubComment("Full logs URL: ${testReportsUrl}")
                     }
+                }
+            }
+
+            node('ansible') {
+                script {
+                    deleteDir()
+                    def logServer = [
```

Comment:
This struct definition is a duplicate from previous step. We only need it for `getLogsURL(logServer, env.ZUUL_UUID)`, in which we only ook at `folder` and `rootDir` fields. Not sure how to get rid of this partial duplication in both steps. Any suggestions?

Reply:
e.g. I could change the signature of `getLogsURL` to `(String rootDir, String folder, String zuulUuid)` and then just call

```
def testReportsUrl = getLogsURL(env.LOG_SERVER, env.LOG_SERVER_FOLDER, env.ZUUL_UUID)
```

but then I can't justify existence of `def logServer` struct.